test/onnx_import.cpp

```diff
@@ -413,3 +413,52 @@ TEST(onnx, model_softmax)
     auto result_vectors = execute(function, inputs, "INTERPRETER");
     EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
 }
+
+TEST(onnx, model_flatten)
+{
+    auto function = onnx_import::import_onnx_function(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/flatten.onnx"));
+
+    std::vector<std::vector<float>> inputs;
```

Comment:
You may use `using Inputs = std::vector<std::vector<float>>;` declared at the beginning of file.